src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -836,15 +836,16 @@ public function unsignedBigInteger($column, $autoIncrement = false)
     }
 
     /**
-     * Create a new unsigned big integer (8-byte) column on the table.
+     * Create a new foreign id column on the table with a specific type.
      *
      * @param  string  $column
+     * @param  string  $type
      * @return \Illuminate\Database\Schema\ForeignIdColumnDefinition
      */
-    public function foreignId($column)
+    public function foreignId($column, $type = 'bigInteger')
```

Comment:
This is a breaking change.

Reply:
How so? I just set the default value.

Reply:
Anyone who is overwriting this method will see their implementation break.

Reply:
Oh didn't notice! Thank you, will making a new PR to master